projects/core/src/lib/gallery.module.ts

```diff
@@ -17,22 +17,15 @@ import { GalleryThumbComponent } from './components/gallery-thumb.component';
 import { GalleryImageComponent } from './components/templates/gallery-image.component';
 import { GalleryVideoComponent } from './components/templates/gallery-video.component';
 import { GalleryIframeComponent } from './components/templates/gallery-iframe.component';
-import { RadialProgressComponent } from './components/templates/radial-progress.component';
 
 import { LazyImage } from './directives/lazy-image';
 import { TapClick } from './directives/tap-click';
-import { CachingInterceptor } from './services/cache.interceptor';
-import { RequestCache, RequestCacheWithMap } from './services/cache.service';
 
 @NgModule({
   imports: [
     CommonModule,
     HttpClientModule
```

Comment:
This module can be removed as it's no longer required. It also means merging #310 is no longer required as this PR will contain every necessary change.

Reply:
@MurhafSousli HttpClientModule is still in the code base? can we remove it?